backend/services/helpers_test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022, Yahoo Inc.
```

Comment:
Year change? I guess I have the advantage of living in the present. I imagine this was correct when it was written. :)

Reply:
Oh man - I would love to live in the present! haha yeah it got outdated so quickly 😂